### PR TITLE
Create DECISIONFILE_ENTRY.md

### PR DESCRIPTION
**Pull request by emckean**
creating a template for Decision File entries that we can link to from our own decision file

### CODE REVIEW

File: DECISIONFILE_ENTRY.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+template format for decision files
+
+DATE in YYYY MMM DD
```
Comment (archaeogeek) [inline DECISIONFILE_ENTRY.md]: Maybe rename DATE PROPOSED in YYYY MMM DD?

File: DECISIONFILE_ENTRY.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+MOTION (mandatory field)
+
+PROPOSER
+
+SUMMARY/DISCUSSION (link to email thread if applicable)
+
+STATUS
+
+DECIDED DATE in YYYY MMM DD
```
Comment (archaeogeek) [inline DECISIONFILE_ENTRY.md]: I'd move this up to just below the date proposed, and for the sake of tidiness rename it DECISION DATE in YYYY MMM DD

File: DECISIONFILE_ENTRY.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+DATE in YYYY MMM DD
+
+MOTION (mandatory field)
+
+PROPOSER
+
+SUMMARY/DISCUSSION (link to email thread if applicable)
+
+STATUS
```
Comment (archaeogeek) [inline DECISIONFILE_ENTRY.md]: For consistency will there be a limited number of options here- eg approved/in progress/rejected/etc?

**Comment by jaredmorgs** [comment]
Is this ready to merge @flicstar @emckean ?

**Comment by camerons** [comment]
This has been superseded by https://github.com/thegooddocsproject/governance/wiki/Decision-Log-2019 and is not required any more.
I'm closing this pull request out.

**Pull request closed without merging**